Resources/public/js/models/extensions/ez-contentinfo-attributes.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-contentinfo-attributes', function (Y) {
+    "use strict";
+    /**
+     * The content info attributes extension
+     *
+     * @module ez-contentinfo-attributes
+     */
+    Y.namespace('eZ');
+
+    /**
+     * Extension providing ContentInfo attributes for Models needing them.
+     *
+     * @namespace eZ
+     * @class ContentInfoAttributes
+     * @extensionfor Y.Base
```

Comment:
actually it's an extension for Y.eZ.RestModel as some attributes use a RestModel method.